pkg/project/project.go

```diff
@@ -186,7 +186,8 @@ func (p *Project) readProjectFile() (*E2e, error) {
 	}
 	projectFile := filepath.Join(dir, p.cfg.Dir, "project.yaml")
 	if _, err := os.Stat(projectFile); os.IsNotExist(err) {
-		return nil, microerror.Mask(err)
+		p.logger.Log("function", "readProjectFile", "level", "info", "message", "optional integration/project.yaml not found, continue build")
+		return &E2e{}, nil
```

Comment:
Zero value of a pointer is usually `nil`.

Reply:
In this case the value can be used down the road, no zero value here.